### PR TITLE
Add button anywhere in checkout onepage (#326)

### PR DESCRIPTION
**Pull request by rus-daniel**

### CODE REVIEW

**Review by mehdichaouch — APPROVED** [review]

File: app/design/frontend/base/default/layout/oyst_oneclick.xml [hunk 1]
```diff
@@ -88,4 +88,17 @@
...
+    <checkout_onepage_index>
+        <block type="oyst_oneclick/oneClick" name="oyst.oneclick.checkout.cart" template="oyst/oneclick/checkout/cart/oneclick.phtml" output="toHtml" />
+        <reference name="head">
+            <block type="core/text" name="cdnjs.oyst-oneclick">
+                <action method="setText" ifconfig="oyst/oneclick/enable">
+                    <text helper="oyst_oneclick/getOneClickJs"/>
+                </action>
+            </block>
+            <action method="addCss" ifconfig="oyst/oneclick/enable"><stylesheet>css/oyst/oneclick.css</stylesheet></action>
+            <action method="addItem"><type>skin_js</type><name>js/oyst/checkout/onepage/oneclick.js</name></action>
```
Comment (mehdichaouch) [inline app/design/frontend/base/default/layout/oyst_oneclick.xml]: Why you didn't add ```ifconfig="oyst/oneclick/enable"```to the ```skin_js``` also?

File: skin/frontend/base/default/js/oyst/checkout/onepage/oneclick.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+                        addButton(button, position);
+                    }
+                });
+            }
+        });
+    };
+}
+
+function addButton(button, index)
+{
```
Comment (mehdichaouch) [inline skin/frontend/base/default/js/oyst/checkout/onepage/oneclick.js]: move the brace to the previous line

File: skin/frontend/base/default/js/oyst/checkout/onepage/oneclick.js [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+    var newParent = document.getElementsByClassName("buttons-set")[index];
+    newParent.appendChild(button);
+}
+
+
```
Comment (mehdichaouch) [inline skin/frontend/base/default/js/oyst/checkout/onepage/oneclick.js]: useless breakline.

**Review by mehdichaouch — CHANGES_REQUESTED** [review]

**Review by mehdichaouch — APPROVED** [review]

**Review by mbouchotoyst — APPROVED** [review]

File: skin/frontend/base/default/js/oyst/checkout/onepage/oneclick.js [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    }
+}
+
+function insertButton() {
+    var placeToAppendButton = document.getElementById("oyst-1click-button-wrapper").getAttribute("data-place-to-append");
+    var BreakException = {};
+
+    try {
+        placeToAppendButton.split(/\s*,\s*/).forEach(function (xpathExpression) {
+            var el = document.evaluate(xpathExpression, document, null, XPathResult.FIRST_ORDERED_NODE_TYPE, null).singleNodeValue;
```
[the remark this answers is not in the document]
Comment (mehdichaouch) [reply skin/frontend/base/default/js/oyst/checkout/onepage/oneclick.js]: Arf :/
Do you have a ninja plan B?

**Review by StefBrito — APPROVED** [review]
Good pour moi

**Pull request merged**